content/FIDO2/index.adoc

```diff
@@ -1,23 +1,23 @@
+FIDO2 is an open authentication standard that consists of the link:https://www.w3.org/TR/webauthn/[W3C Web Authentication API] (WebAuthn), and the link:https://fidoalliance.org/specifications/download/[FIDO2 Client to Authentication Protocol] (CTAP). CTAP is an application layer protocol used for communication between a client (browser) or a platform (operating system) and an external authenticator (YubiKey 5). Yubico, Microsoft and Google are the core contributors to the W3C WebAuthn and FIDO2 CTAP2 specifications.
 
-Like link:https://developers.yubico.com/U2F/[FIDO U2F], the FIDO2 standard offers the same high level of security, as it is based on public key cryptography and is intended to solve additional user scenarios including first factor (passwordless) and multi-factor authentication. With these new capabilities, the hardware security key can entirely replace weak static username/password credentials with strong hardware-backed public/private-key credentials.  These credentials can not be reused, replayed or shared across services, and are not subject to phishing and MiTM attacks or server breaches.
+Like U2F, the FIDO2 standard offers the same high level of security, as it is based on public key cryptography and is intended to solve additional user scenarios including first factor (passwordless) and multi-factor authentication. With these new capabilities, the hardware security key can entirely replace weak static username/password credentials with strong hardware-backed public/private-key credentials.  These credentials can not be reused, replayed or shared across services, and are not subject to phishing and MiTM attacks or server breaches.
 
-=== FIDO2 Advantages
+=== FIDO2/WebAuthn Advantages
 
 [horizontal]
 Strong security:: Replaces weak passwords with strong hardware-based authentication using public key crypto to protect against phishing, session hijacking, man-in-the-middle, and malware attacks.
-Privacy Protection:: A FIDO2 device generates a new pair of keys for every service, and only the service stores the public key. With this approach, no secrets are shared between service providers.
+Privacy Protection:: A FIDO2 authenticator generates a new pair of keys for every service, and only the service stores the public key. With this approach, no secrets are shared between service providers.
```

Comment:
Maybe: s/only the service stores the public key/the service stores only the public key/ ?

Reply:
incorporated into commit eacd866